Assets/_BForBoss/_Core/Scripts/Debug/DebugWindow.cs

```diff
@@ -25,47 +25,35 @@ public class DebugWindow : MonoBehaviour
         
         private bool _isPanelShowing = false;
         
-        private List<Type> _debugOptions = new List<Type>();
+        private List<DebugView> _debugViews = new List<DebugView>();
```

Comment:
I'm storing the views instead of recreating the views every frame.  Takes a toll on my laptop,  and some memory leaks is exacerbated on my laptop